src/diffusers/configuration_utils.py

```diff
@@ -144,6 +144,12 @@ def save_config(self, save_directory: Union[str, os.PathLike], push_to_hub: bool
         Args:
             save_directory (`str` or `os.PathLike`):
                 Directory where the configuration JSON file is saved (will be created if it does not exist).
+            push_to_hub (`bool`, *optional*, defaults to `False`):
+                Whether or not to push your model to the Hugging Face Hub after saving it. You can specify the
+                repository you want to push to with `repo_id` (will default to the name of `save_directory` in your
+                namespace).
+            kwargs (`Dict[str, Any]`, *optional*):
+                Additional keyword arguments passed along to the [`~utils.PushToHubMixin.push_to_hub`] method.
```

Comment:
Since we are not calling `push_to_hub` but `create_repo` and `_upload_folder` independently, I think we should remove the `kwargs` parameter. For example if `create_pr` is passed as kwarg, it will be passed to `create_repo` which will fail.